stock_picking_variant_mgmt/wizard/stock_manage_variant_view.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="stock_manage_variant_form">
+        <field name="model">stock.manage.variant</field>
+        <field name="arch" type="xml">
+            <form string="Manage Product Variants">
+                <group>
+                    <field name="product_tmpl_id"
+                           domain="[('attribute_line_ids', '!=', False)]"
+                           options="{'no_create': True}"
+                    />
+                </group>
+                <field name="variant_line_ids"
```

Comment:
Put this field inside a group with nolabel, now the field appears in small area.